platforms/solana/src/chain.ts

```diff
@@ -3,27 +3,27 @@ import {
   ChainContext,
   NativeAddress,
   UniversalAddress,
-  UniversalOrNative,
-  Platform,
   toNative,
 } from '@wormhole-foundation/connect-sdk';
 import { getAssociatedTokenAddress } from '@solana/spl-token';
 import { SolanaPlatform } from './platform';
+import { AnySolanaAddress } from './types';
+import { SolanaAddress } from './address';
 
 export class SolanaChain extends ChainContext<'Solana'> {
   // @ts-ignore
   readonly platform = SolanaPlatform;
 
   async getTokenAccount(
-    token: UniversalOrNative<'Solana'> | 'native',
+    token: AnySolanaAddress | 'native',
     address: UniversalAddress,
   ): Promise<NativeAddress<'Solana'>> {
     const tb = await this.getTokenBridge();
 
-    const mintAddress: UniversalOrNative<'Solana'> =
+    const mintAddress =
       token === 'native'
         ? await tb.getWrappedNative()
-        : token.toUniversalAddress();
+        : new SolanaAddress(token);
```

Comment:
This gets called from the wormhole class to prep a transfer, I'm not positive but it _may_ cause issues if the address passed has not been "universal-ified"

Reply:
Ah good point, I'll revert for now.  Maybe in the future, we should just throw an error if they try to send to Solana but don't have the Solana package installed.  Just add some more safety guardrails

Reply:
good idea re error on no Solana, hoping this specific case can be eliminated once the new Solana contracts go to Mainnet since it allows sending to standard acct not ATA